documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -544,30 +546,503 @@ endif::community[]
+Enter one of the following commands:
++
+[source,shell,subs="+quotes"]
+----
+podman push _<myregistry.io>_/debezium-jdbc-connector-container:latest
+----
++
+[source,shell,subs="+quotes"]
+----
+docker push _<myregistry.io>_/ddebezium-jdbc-connector-container:latest
+----
+
+.. Create a new {prodname} Oracle KafkaConnect custom resource (CR).
+For example, create a `KafkaConnect` CR with the name `dbz-connect.yaml` that specifies `annotations` and `image` properties.
+The following example shows an excerpt from a `dbz-connect.yaml` file that describes a `KafkaConnect` custom resource. +
++
+=====================================================================
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: debezium-kafka-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true" // <1>
+spec:
+  image: debezium-jdbc-connector-container // <2>
+
+  ...
+----
+=====================================================================
+
+[cols="1,7",options="header"]
+|===
+|Item |Description
+
+|1
+|You must set `metadata.annotations` to `true` to permit the Cluster Operator to use `KafkaConnector` resources to configure {prodname} connectors in the cluster.
+
+|2
+|`spec.image` specifies the name of the image that you created to run your {prodname} connector.
+This property overrides the `STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE` variable in the Cluster Operator.
+
+|===
+
+.. Apply the `KafkaConnect` CR to the OpenShift Kafka Connect environment by entering the following command:
++
+[source,shell,options="nowrap"]
+----
+oc create -f dbz-connect.yaml
+----
++
+The command adds a Kafka Connect instance that specifies the name of the image that you created to run your {prodname} connector.
+
+. Create a `KafkaConnector` custom resource that configures your {prodname} JDBC connector instance.
++
+You configure a {prodname} JDBC connector in a `.yaml` file that specifies the configuration properties for the connector.
++
+The following example shows an excerpt from a `dbz-connect.yaml` file that sets a few of the key properties for a `KafkaConnect` custom resource. +
+The connector establishes a JDBC connection to a PostgreSQL server sink on port `5432`. +
+ +
+For information about the full range of available connector properties, see xref:debezium-jdbc-connector-descriptions-of-connector-configuration-properties[Descriptions of {prodname} JDBC connector configuration properties].
++
+.`jdbc-connector.yaml`
+=====================================================================
+[source,yaml,subs="+attributes,+quotes",options="nowrap"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnector
+metadata:
+  labels:
+  strimzi.io/cluster: my-connect-cluster // <1>
+  name: orders-topic-to-postgresql-via-jdbc-sink-connector // <2>
+...
+spec:
+  class: io.debezium.connector.jdbc.JdbcSinkConnector // <3>
```

Comment:
```suggestion
  class: io.debezium.connector.jdbc.JdbcSinkConnector // <3>
```
As with L757, I suspect that this `spec.class` entry should be a `config.connector.class` entry.

Reply:
Just to be sure, I just checked the `KafkaConnector` CRD yaml, and according to [this](https://github.com/strimzi/strimzi-kafka-operator/blob/main/install/cluster-operator/047-Crd-kafkaconnector.yaml), you need `spec.class` but you don't or would use `config.connector.class`.

Reply:
> Just to be sure, I just checked the `KafkaConnector` CRD yaml, and according to [this](https://github.com/strimzi/strimzi-kafka-operator/blob/main/install/cluster-operator/047-Crd-kafkaconnector.yaml), you need `spec.class` but you don't or would use `config.connector.class`.

Thanks, @Naros. So, I think that part of the confusion surround this is that we list the following property in all of the source connector docs: `connector.class`, for example, this instance in the Db2 doc: https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/db2.adoc#L2443
Also, in the JDBC doc the existing upstream example includes a reference to `connector.class`:  https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/jdbc.adoc#L617

Reply:
Is my understanding of what should be there correct? We only need `spec.class`, right? 

I was thrown off by what might be a typo when you said, 

> ... you need `spec.class` but you don't _or would use_ `config.connector.class`. [emphasis mine]

Is it that you would use `config.connector.class` if you don't specify `spec.class`?

Reply:
So I double checked with Jakub from Strimzi and verified this also in the [docs](https://strimzi.io/docs/operators/latest/configuring.html#type-KafkaConnectorSpec-reference).

According to the docs, the `config` property is a `map` and it says:
> The Kafka Connector configuration. The following properties cannot be set: name, connector.class, tasks.max.

Perhaps in older Strimzi versions this wasn't the case, but at least based on 0.45, the documentation discourages and claims those shouldn't and cannot be set.

Reply:
Ah, ok. So I'm guessing that where we specify `config.connect.class` in the community version of the instructions, that works, because in that context Strimzi is not part of the environment, yes?

Reply:
To prevent confusion, for now, I can conditionalize the `connector.class` entries in the properties tables to apply to the community version only. WDTY?